libs/datamodel/core/src/validate/validation_pipeline/validations/models.rs

```diff
@@ -270,3 +270,29 @@ pub(super) fn id_client_name_does_not_clash_with_field(model: ModelWalker<'_>, c
         ));
     }
 }
+
+pub(super) fn schema_exists(model: ModelWalker<'_>, ctx: &mut Context<'_>) {
+    let (_schema, span) = match (model.schema(), ctx.datasource) {
+        (Some((schema_name, span)), Some(ds)) if !ds.has_schema(schema_name) => (schema_name, span),
+        _ => return,
+    };
+    ctx.push_error(DatamodelError::new_static(
+        "This schema is not defined in the datasource. Read more on `@@schema` at https://pris.ly/multi-schema",
```

Comment:
Docs links always have a `/d/`:
```suggestion
        "This schema is not defined in the datasource. Read more on `@@schema` at https://pris.ly/d/multi-schema",
```